### ISSUE TITLE
bug with dot-notation

### ISSUE THREAD

**Issue by evilebottnawi** [eslint/eslint#2853]
Node.js, gulp
```javascript
.pipe($.if('*.png', gulp.dest(directory.destSprites, {
    verbose: cli.verbose
}), gulp.dest(directory.destLess, {
    verbose: cli.verbose
})))
```
$ - alias for module https://github.com/jackfranklin/gulp-load-plugins
if - module https://github.com/robrich/gulp-if

**Comment by lo1tuma** [comment]
I don’t understand what kind of bug you are reporting here. What does the `dot-notation` rule report for you example and what would you expect instead?

**Comment by evilebottnawi** [comment]
@lo1tuma here not error. Rule think that if is keywork, but if is function module

**Comment by lo1tuma** [comment]
What is your rule configuration?

**Comment by evilebottnawi** [comment]
@lo1tuma 
```json
"dot-notation": [
    2, {
        "allowKeywords": false
    }
]
```

**Comment by lo1tuma** [comment]
According to your configuration you explicitly disallow keywords to be used with the dot notation, so the rule warns correctly on `if` because it is a reserved word. If you want to change this you have to set `"allowKeywords"` to `true`.

**Comment by evilebottnawi** [comment]
@lo1tuma thanks, sorry that took the time
